src/main/java/com/jcabi/github/mock/MkIssue.java

```diff
@@ -176,7 +177,20 @@ public int compareTo(
     public void patch(
         @NotNull(message = "json can't be NULL") final JsonObject json
     ) throws IOException {
+        final Issue.Smart smart = new Issue.Smart(this);
+        final boolean wasopen = smart.isOpen();
         new JsonPatch(this.storage).patch(this.xpath(), json);
+        final boolean nowopen = smart.isOpen();
```

Comment:
@cvrebert let's avoid complex variable names and rename it to something like `now`